### ISSUE TITLE
Paragraph breaks function

### ISSUE THREAD

**Issue by mbuscemi** [djc/askama#464]
My project required me to create the following filter:

```
mod filters {
    pub fn paragraphbreaks(s: &str) -> ::askama::Result<String> {
        let s = s.to_string();
        let linebroken = s.replace("\n\n", "</p><p>");
    
        Ok(format!("<p>{}</p>", linebroken))
    }
}
```

Essentially, it is the linebreaks filter, but without converting single newlines to br tags. It seems like this would have a pretty wide application across projects. I would be happy to generate a pull request if you would like it in Askama.

**Comment by djc** [comment]
Hmm, I think this could make sense. @vallentin any opinions?

**Comment by vallentin** [comment]
Yeah, it's a nice filter. I just have one question. Should there be any special behavior for more than 2 consecutive `\n`s? E.g. `\n\n\n`, `\n\n\n\n`, `\n\n\n\n\n`, etc.

It would also be nice, if a possible implementation supported `\r\n\r\n`. (Along with adding that support to `linebreaks` and `lienbreaksbr`.)

**Comment by djc** [comment]
What kind of special behavior for multiple consecutive newlines would you have in mind? I can't think of any one behavior that especially makes sense and is also straightforward to express in code.

**Comment by vallentin** [comment]
I was more thinking whether e.g. `Foo\n\n\n\nBar` should produce `<p>Foo</p><p></p><p>Bar</p>` (the current filter) or `<p>Foo</p><p>Bar</p>`. If the second result is more desirable, then adding `.replace("<p></p>", "")` after the first replace, would be a straight forward solution.

**Comment by djc** [comment]
@mbuscemi what do you think?

**Comment by mbuscemi** [comment]
Sure! I like the addition. I'll put a PR together.

**Comment by mbuscemi** [comment]
PR is here: https://github.com/djc/askama/pull/467
